gcp_deepvariant_runner.py

```diff
@@ -256,25 +314,34 @@ def _is_valid_gcs_path(gcs_path):
   Args:
     gcs_path: (str) a path to directory or an obj on GCS.
   """
-  return urlparse.urlparse(gcs_path).scheme == 'gs'
+  return (urlparse.urlparse(gcs_path).scheme == 'gs' and
+          urlparse.urlparse(gcs_path).netloc)
 
 
-def _gcs_object_exist(gcs_obj_path):
-  """Returns true if the given path is a valid object on GCS.
+def _get_gcs_object_size(gcs_obj_path):
+  """Returns the size of the given GCS object.
 
   Args:
     gcs_obj_path: (str) a path to an obj on GCS.
   """
   try:
-    storage_client = storage.Client()
     bucket_name = _get_gcs_bucket(gcs_obj_path)
     obj_name = _get_gcs_relative_path(gcs_obj_path)
-    bucket = storage_client.bucket(bucket_name)
-    obj = bucket.blob(obj_name)
-    return obj.exists()
-  except google_exceptions.Forbidden as e:
-    logging.error('Missing GCS object: %s', str(e))
-    return False
+  except ValueError as e:
+    logging.error('Invalid GCS path: %s', str(e))
+    return 0
+
+  storage_client = storage.Client()
+  try:
+    bucket = storage_client.get_bucket(bucket_name)
+  except (google_exceptions.NotFound, google_exceptions.Forbidden) as e:
+    logging.error('Unable to access GCS bucket: %s', str(e))
+    return 0
+
+  blob = bucket.get_blob(obj_name)
```

Comment:
Does this raise exception? When a null blob is returned? 
We had it in try block, but now it's not guarded.